plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSourceProvider.java

```diff
@@ -111,6 +115,11 @@ public ConnectorPageSource createPageSource(
         DeltaLakeSplit split = (DeltaLakeSplit) connectorSplit;
         DeltaLakeTableHandle table = (DeltaLakeTableHandle) connectorTable;
 
+        List<DeltaLakeColumnHandle> deltaLakeColumns = columns.stream()
```

Comment:
What is the point of moving it further away from place where the variables are used?